dbx/encrypt-fields.rst

```diff
@@ -11,58 +11,79 @@ In-Use Encryption
 Overview
 --------
 
-You can encrypt fields in a document using a set of features called
-**in-use encryption**.
-
-In-use encryption enables your client applications to encrypt data
-*before* sending it to MongoDB, and to query documents with encrypted fields.
-
-Because the driver encrypts the data before sending it to MongoDB, only
-your configured client applications can decrypt the data. Only applications
-using the driver with access to your encryption keys can access the decrypted,
-plaintext data. Should you have unauthorized access to your database, an
-attacker could only see the encrypted, ciphertext data.
-
-In-use encryption can help prevent exposure of the following sensitive types of data:
+You can use the {+driver-short+} to encrypt specific document fields by using a
+set of features called **in-use encryption**. In-use encryption allows
+your application to encrypt data *before* sending it to MongoDB
+and query documents with encrypted fields.
+
+In-use encryption prevents unauthorized users from viewing plaintext
+data as it is sent to MongoDB or when it is in an encrypted database. To
+enable in-use encryption in an application and authroize it to decrypt
+data, you must create encryption keys that only your application has
+access to. Only these applications that have access to your encryption
+keys can access the decrypted, plaintext data. This means that in the
+event of a security breach, a potential attacker sees only encrypted,
+ciphertext data as they do not have access to the encryption keys.
+
+You might use in-use encryption to encrypt fields in your MongoDB
+documents that contain the following types of sensitive data:
 
 - Credit card numbers
 - Addresses
 - Health information
 - Financial information
 - Any other sensitive or personally identifiable information (PII)
 
-MongoDB offers the following ways to encrypt fields:
+MongoDB offers the following features to enable in-use encryption:
+
+- :ref:`Queryable Encryption <subsection-qe>`
+- :ref:`Client-side Field Level Encryption <subsection-csfle>`
+
+.. _subsection-qe:
 
 Queryable Encryption
 ~~~~~~~~~~~~~~~~~~~~
 
 Queryable Encryption is the next-generation in-use encryption feature,
-introduced in MongoDB Server version 6.0 and available as a public
-preview. Queryable Encryption supports searching encrypted fields for
-equality and encrypts each value uniquely.
+first introduced as a preview feature in MongoDB Server version 6.0 and
+as a generally available (GA) feature in MongoDB 7.0. Queryable
```

Comment:
Suggestion:

I think (GA) isn't needed since the abbreviation isn't used again.

Reply:
It is used in the admonition below this paragraph!